src/test.jl

```diff
@@ -118,7 +149,9 @@ function _test(T, name::Symbol, condition, obj, i=nothing)
     obj_copy = deepcopy(obj)
     res = try
         f = condition isa Pair ? condition[2] : condition
+        # GC.enable(false)
```

Comment:
What's up with that?

Reply:
Yeah, remnant of tryng to fix `WeakRefDict` - GC collects the refs before we can test with them.